platform/ios/src/MGLMapView.mm

```diff
@@ -774,6 +784,31 @@ - (UIViewController *)viewControllerForLayoutGuides
 
 - (void)updateConstraints
 {
+    // scale control
+    //
+    [self removeConstraints:self.scaleControlViewConstraints];
+    [self.scaleControlViewConstraints removeAllObjects];
+    
+    [self.scaleControlViewConstraints addObject:
+     [NSLayoutConstraint constraintWithItem:self.scaleBarView
+                                  attribute:NSLayoutAttributeTop
+                                  relatedBy:NSLayoutRelationEqual
+                                     toItem:self
+                                  attribute:NSLayoutAttributeTop
+                                 multiplier:1
+                                   constant:5+self.contentInset.top]];
```

Comment:
We've been using 5 on the compassView for quite a while. Let's revisit when we replace these constraints in #7716